rust/src/smb/smb.rs

```diff
@@ -1885,22 +1884,26 @@ impl SMBState {
     pub fn parse_tcp_data_ts_gap(&mut self, gap_size: u32) -> AppLayerResult {
         SCLogDebug!("GAP of size {} in toserver direction", gap_size);
         let consumed = self.handle_skip(Direction::ToServer, gap_size);
+        if consumed == gap_size {
```

Comment:
Should be squashed into the first commit changing this ;-)

Reply:
Done